src/codegen/enums.rs

```diff
@@ -396,7 +396,8 @@ impl FromGlib<{sys_crate_name}::{ffi_name}> for {name} {{
     unsafe fn from_value(value: &'a {gvalue}) -> Self {{
         {assert}from_glib({glib}(value.to_glib_none().0))
     }}
-}}",
+}}
+            unsafe impl FromOwnedValue for {name} {{}}",
```

Comment:
The indent is completely broken here. It should be:

```suggestion
unsafe impl FromOwnedValue for {name} {{}}",
```

Reply:
Does it really matter ? At the end we call rustfmt to fix the broken formatting

Reply:
It does, the code didn't make sense when I was reading it because I thought it was an impl inside an impl (because of the indent).

Reply:
Fair, updated